cmd/aspell-dictionaries.rb

```diff
@@ -22,13 +22,12 @@ def run
         dictionary_mirror = "https://ftpmirror.gnu.org/aspell/dict"
         languages = {}
 
-        index_output, = Utils::Curl.curl_output("#{dictionary_url}/0index.html")
+        index_output = Utils::Curl.curl_output("#{dictionary_url}/0index.html").stdout
         index_output.split("<tr><td>").each do |line|
           next unless line.start_with?("<a ")
 
-          _, language, _, path, = line.split('"')
-          language.tr!("-", "_")
-          languages[language] = path
+          _, language, _, _path = line.split('"')
+          T.must(language).tr!("-", "_")
         end
 
         resources = languages.map do |language, path|
```

Comment:
I haven't looked at this command deeply but, unless I'm missing something, we have to store the path in `languages` to be able to use it below in the `languages.map` call. Without `languages[language] = path`, `languages` will remain empty, so `resources` will be empty and all of that logic basically gets skipped.

Reply:
yeah, that is right, updated the logic.